libfabric.spec

```
@@ -1,10 +1,8 @@
 %define suse_libname libfabric1
-#global dl_version 1.11.1rc1
-
-%{?dl_version:%global autosetup_args --p1 -n libfabric-%{dl_version}}
+%global dl_version 1.12.0rc1
 
 Name: libfabric
-Version: 1.11.1
+Version: 1.12.0~rc1
```

Next we define the version from the components:
```suggestion
Version: 1.12.0~rc1
Version: %{major}.%{minor}.%{bugrelease}%{?prerelease:~%{prerelease}}
```